doc/prototype/filter-members.md

```diff
@@ -5,46 +5,64 @@
 
 @signature `queryLogic.filterMembers(a, [b,] bData)`
 
-Filters the `a` query's records from another _super set_ query's records.
-
-```js
-queryLogic.filterMembers(
-  {filter: {type: "dog"}},
-  {filter: {type: ["dog", "cat"]}},
-  [{id: 1, type:"cat"},
-   {id: 2, type: "dog"},
-   {id: 3, type: "dog"},
-   {id: 4, type: "zebra"}]
-) //-> [{id: 2, type: "dog"},{id: 3, type: "dog"}]
-```
-
-The _super set_ `b` argument is optional.  It defaults to assuming the universal query.  For example,
-the following still works:
-
-```js
-queryLogic.filterMembers(
-  {filter: {type: "dog"}},
-  [{id: 1, type:"cat"},
-   {id: 2, type: "dog"},
-   {id: 3, type: "dog"},
-   {id: 4, type: "zebra"}]
-) //-> [{id: 2, type: "dog"},{id: 3, type: "dog"}]
-```
-
-The _super set_ `b` argument is important when pagination is present:
-
-```js
-queryLogic.filterMembers(
-  {page: {start: 11, end: 12}},
-  {page: {start: 10, end: 13}},
-  [{id: 1, type:"cat"},
-   {id: 2, type: "dog"},
-   {id: 3, type: "dog"},
-   {id: 4, type: "zebra"}]
-) //-> [{id: 2, type: "dog"},{id: 3, type: "dog"}]
-```
-
-records will be returned sorted by the query's `sort` property.
+  Filters the `a` query's records from another _super set_ query's records.
+
+  ```js
+  import {QueryLogic} from "can";
+
+  const queryLogic = new QueryLogic();
+
+  const filter = queryLogic.filterMembers(
+    {filter: {type: "dog"}},
+    {filter: {type: ["dog", "cat"]}},
+    [{id: 1, type:"cat"},
+    {id: 2, type: "dog"},
+    {id: 3, type: "dog"},
+    {id: 4, type: "zebra"}]
+  );
+  console.log( filter ); //-> [{id: 2, type: "dog"},{id: 3, type: "dog"}]
+  ```
+  @codepen
+
+  The _super set_ `b` argument is optional.  It defaults to assuming the universal query.  For example,
+  the following still works:
+
+  ```js
+  import {QueryLogic} from "can";
+
+  const queryLogic = new QueryLogic();
+
+  const filter = queryLogic.filterMembers(
+    {filter: {type: "dog"}},
+    [{id: 1, type:"cat"},
+    {id: 2, type: "dog"},
+    {id: 3, type: "dog"},
+    {id: 4, type: "zebra"}]
+  );
+  console.log( filter ); //-> [{id: 2, type: "dog"},{id: 3, type: "dog"}]
+  ```
+  @codepen
+
+  The _super set_ `b` argument is important when pagination is present:
```

Comment:
Technically, it's only really important if the parent set is paginated.  Might want to talk about this and explain how the example after this is demonstrating that.